portingdb/templates/howto.html

```diff
@@ -0,0 +1,156 @@
+        </p><p>
+            Now open the package in PortingDB (e.g. {{ pkglink(random_mispackaged) }}): The {{ badge(mispackaged_status) }} next to <i>Fedora</i> in the sidebar indicates that indeed we are dealing with a mispackaged package.
+        </p>
+        <h3>What's What</h3>
+        <p>
+            Look further down the sidebar for the headline <i>RPMs</i>. There you will see all the RPM (sub)packages contained in this package. Sometimes there's just the base package, sometimes it has several subpackages. What we want to know is if there is a (sub)package for Python 3. If there is only a Python 2 (sub)package, then we know that this package needs to be ported.
+        </p><p>
+            However, sometimes you will find a package that is still mispackaged even though it already has a Python 3 (sub)package. In those cases, something isn't quite right. Usually the Python 3 subpackage depends on both Python 3 <i>and</i> Python 2. That is unfortunate, because it basically renders the porting effort null.
+        </p>
+        <h3>Make a Patch</h3>
+        <p>
+            In either case, look a bit further down still and find a link to the associated Bugzilla report (under <i>Links→bug</i> in the sidebar). Go there and join the discussion. Find out what's the status quo and if anyone is working on it. If not, be the one that submits a patch!
+            <!-- TODO: Link for How to submit/create a patch? -->
+        </p><p>
+            To aid you in your porting effort, use the <a href="http://python-rpm-porting.readthedocs.io">Python RPM Porting Guide</a>. If you are investigating why a Python 3 subpackage still depends on Python&nbsp;2, take a look at the <a href="http://python-rpm-porting.readthedocs.io/en/latest/application-modules.html#are-shebangs-dragging-you-down-to-python-2">section on <i>shebangs</i></a>.
+        </p>
+        <h3>Make it Happen</h3>
+        <p>
+            When you submit your patch to the associated Bugzilla report (use the <i>"Add an attachment"</i> button), be sure to ask the assgned packager to review and push it. However, as maintainers can be unresponsive for long periods of time, let us know about your patch as well so we can push it if the maintainer is unable.
+        </p><p>
+            To let us know, edit the <i>Blocks</i> field of the Bugzilla report and add the string <b>PY3PATCH-AVAILABLE</b> (you can use space as a separator). That way it will appear in our <a href="https://bugzilla.redhat.com/show_bug.cgi?id=1312032">tracking bug</a> for packages with an available patch. If need be, we'll review your patch and push it for the maintainer.
+        </p>
+        <h3>Fame and Glory</h3>
+        <p>
+            After you've done the deed, you deserve your reward! Sadly, the hand of the King's daughter and half of all the kingdom has recently become unavailable (may vary upon location, ask your nearest royal family), but fret not, you can get a <i>badge</i> of honour!
```

Comment:
It's not only Czech, but also anglophone (I checked). And if someone misses it, I think they can read on.